platformio/commands/account/command.py

```diff
@@ -0,0 +1,198 @@
+
+
+@click.group("account", short_help="Manage PIO Account")
+def cli():
+    pass
+
+
+def validate_username(value):
+    value = str(value).strip()
+    if not re.match(r"^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){3,38}$", value, flags=re.I):
+        raise click.BadParameter(
+            "Invalid username format. "
+            "Username must contain at least 4 characters including single hyphens,"
+            " and cannot begin or end with a hyphen"
+        )
+    return value
+
+
+def validate_email(value):
+    value = str(value).strip()
+    if not re.match(r"^[a-z\d_.+-]+@[a-z\d\-]+\.[a-z\d\-.]+$", value, flags=re.I):
+        raise click.BadParameter("Invalid E-Mail address")
+    return value
+
+
+def validate_password(value):
+    value = str(value).strip()
+    if not re.match(r"^(?=.*[a-z])(?=.*\d).{8,}$", value):
+        raise click.BadParameter(
+            "Invalid password format. "
+            "Password must contain at least 8 characters"
+            " including a number and a lowercase letter"
+        )
+    return value
+
+
+@cli.command("register", short_help="Create new PIO Account")
+@click.option(
+    "-u",
+    "--username",
+    prompt=True,
+    callback=lambda _, __, value: validate_username(value),
+)
+@click.option(
+    "-e", "--email", prompt=True, callback=lambda _, __, value: validate_email(value)
+)
+@click.option(
+    "-p",
+    "--password",
+    prompt=True,
+    hide_input=True,
+    confirmation_prompt=True,
+    callback=lambda _, __, value: validate_password(value),
+)
+@click.option("--first-name", prompt=True)
+@click.option("--last-name", prompt=True)
+def account_register(username, email, password, first_name, last_name):
+    client = AccountClient()
+    try:
+        client.registration(username, email, password, first_name, last_name)
+        return click.secho(
+            "An account has been successfully created. "
+            "Please check your mail to activate your account and verify your email address.",
+            fg="green",
+        )
+    except exception.AccountAlreadyAuthenticated as e:
+        return click.secho(str(e), fg="yellow",)
+
+
+@cli.command("login", short_help="Log in to PIO Account")
+@click.option("-u", "--username", prompt="Username or e-mail")
+@click.option("-p", "--password", prompt=True, hide_input=True)
+def account_login(username, password):
+    client = AccountClient()
+    try:
+        client.login(username, password)
+        return click.secho("Successfully logged in!", fg="green")
+    except exception.AccountAlreadyAuthenticated as e:
+        return click.secho(str(e), fg="yellow",)
+
+
+@cli.command("logout", short_help="Log out of PIO Account")
+def account_logout():
+    client = AccountClient()
+    try:
+        client.logout()
+        return click.secho("Successfully logged out!", fg="green")
+    except exception.AccountNotAuthenticated as e:
+        return click.secho(str(e), fg="yellow",)
+
```

Comment:
I can't logout when token is broken. If token is broken or is not valid, JUST REMOVE data from state
```
$ pio account logout
Error: Invalid refresh token
```